docs/index.rst

```diff
@@ -1,3 +1,5 @@
+.. _main-index:
+

```

Comment:
In addition to the fixes, I added a new link, to return to the index when needed.